Library/Formula/generate-json-schema.rb

```diff
@@ -15,10 +15,23 @@ class GenerateJsonSchema < Formula
 
   depends_on "node"
 
+  resource "npm-type-of-is" do
+    url "https://registry.npmjs.org/type-of-is/-/type-of-is-3.4.0.tgz"
+    sha256 "5e09fc1d5968ef748ecac3d048244cc2a28ef4e6b503342662d0734498d166b8"
+  end
+
   def install
-    ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+    npm = "#{Formula["node"].opt_libexec}/npm/bin/npm"
+    # let us manage the dependencies
+    inreplace "package.json",
+      /,\n  "dependencies": \{[^}]+\},\n  "devDependencies": \{[^}]+\}/, ""
```

Comment:
Tempted to just not vendor for now. We should probably have a larger issue to discuss this stuff as I'm not convinced it's worth the effort for bottled deps.